fastlane_core/lib/fastlane_core/itunes_transporter.rb

```diff
@@ -51,6 +51,33 @@ def build_verify_command(username, password, source = "/tmp", provider_short_nam
       not_implemented(__method__)
     end
 
+    # Runs preparations before executing any command from the executor.
+    #
+    # @param [Hash] api key containing the issuer id and private key
+    # @return [Hash] copy of `api_key` which includes an extra `key_dir` with the location of the .p8 file on disk
+    def prepare(original_api_key)
```

Comment:
Nitpick: I feel like it'd help readability (including at call site) to use keyword parameter here?
```suggestion
    def prepare(original_api_key:)
```
Then update all the call sites with `@transporter_executor.prepare(original_api_key: @api_key)`

Reply:
updated ✔️